dev-build/conf/logback.xml

```diff
@@ -29,7 +29,11 @@
 
     <root level="INFO">
         <appender-ref ref="LOGFILE"/>
-        <appender-ref ref="STDOUT"/>
+        <if condition='!property("STAGE").contains("DEV")'>
+            <then>
+                <appender-ref ref="ASYNCSTDOUT"/>
+            </then>
+        </if>
```

Comment:
I hadn't seen this approach (using [Conditional processing & Janino](https://logback.qos.ch/setup.html#janino)) for having different PROD & DEV logging configuration before, but doing a [GitHub search](https://github.com/search?q=org%3Aguardian+org.codehaus.janino&type=code) I can see it's being used in 4 other places elsewhere at the Guardian, presumably successfully?

The approach I was more familiar with, from Ophan, was having two separate logback files:

* [`logback.xml`](https://github.com/guardian/ophan/blob/595b66ef1609e7f42be01339cb94dab603c3abf7/dashboard/conf/logback.xml) - this gets used in dev
* [`logback-PROD.xml`](https://github.com/guardian/ophan/blob/595b66ef1609e7f42be01339cb94dab603c3abf7/dashboard/conf/logback-PROD.xml)

...then we take advantage of the fact that when we create Production artifacts, we can [specify custom `javaOptions`](https://github.com/guardian/ophan/blob/595b66ef1609e7f42be01339cb94dab603c3abf7/build.sbt#L372-L373) in `build.sbt`, adding a `-Dlogger.resource=logback-PROD.xml` argument  - so only built artifacts get run with that parameter, not when you're running the code locally in dev.

Like I say, I wasn't familiar with the _Conditional processing & Janino_ approach, but I think both approaches have advantages/disadvantages.

Does `property("STAGE")` mean the _environment variable_ or the _system property_ - and what is actually set in our running Frontend instances?